src/Actions.js

```diff
@@ -35,7 +35,7 @@ export function create({ name, map }) {
   function action(value) {
     let err = null;
     try {
-      value = map(value);
+      value = map.bind(this)(value);
     } catch(e) {
```

Comment:
Lets move the bind of map outside the action object and pass in the instance object as an argument to `createMany` on line 92 and within `createMany` the instance object will be used as the second argument to `this.map` on line 81. That will make the context of `this === actionsObj` within `create` function.